### DISCUSSION TITLE
PHP lexer minor bug

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [orbitalquark/textadept#311] [Q&A] [answered]
It doesn't highlight the variables. On PHP, variable starts with the `$` sign. Both the variable and the `$` sign should be highlighted. Currently, none of them is highlighted.

**Comment by orbitalquark** [comment, marked as answer]
This is not actually a bug because the lexer is recognizing and tagging $ words as variables (Tools > Show Style). The default theme is just not applying a different color to them. You can use a different theme to highlight variables in a different color if you'd like.

**Reply by nsgnkhibdk2cls0f** [reply]
Which theme? The dark theme also doesn't highlight them. As the document said, there are only light, dark and term theme:

https://orbitalquark.github.io/textadept/manual.html#themes

**Comment by orbitalquark** [comment]
I don't have a particular theme in mind. You are right that the default themes that ship with Textadept do not highlight variables. You could modify Textadept's default themes or create your own local copy with the changes you desire.

**Reply by nsgnkhibdk2cls0f** [reply]
This is too far from my original purpose. Knowing the lexer actually recognizing it is enough for now.
